src/basic-syntax/rustdoc.md

```diff
@@ -30,4 +30,7 @@ idiomatic to document all public items in an API using this pattern.
 * Inner doc comments are discussed later (in the page on modules) and need not
   be addressed here.
 
+* Rustdoc comments can contain code snippets that we can run and test using `cargo test`.
+  We will discuss these tests in the [Testing section](../testing/doc-tests.html).
```

Comment:
Just a small note, we can (and should) link to the Markdown files instead. That makes the links work both in GitHub and in the rendered HTML (`mdbook` will replace `.md` with `.html` in the link).